LiskHQ/lisk/test.sh fails to run tests

### Expected behavior

`LiskHQ/lisk/test.sh` runs the test suite successfully with the `NODE_ENV` environment variable set.

### Actual behavior

- `LiskHQ/lisk/test.sh` fails with an error when attempting to stop any existing application process.

```
Running untagged all tests...
[PM2][ERROR] Process app.js not found

npm ERR! Darwin 17.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "test" "untagged" "all"
npm ERR! node v6.12.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! lisk@0.9.10 test: `bash test.sh "untagged" "all"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lisk@0.9.10 test script 'bash test.sh "untagged" "all"'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lisk package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bash test.sh "untagged" "all"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lisk
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lisk
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Oliver/GitHub/LiskHQ/lisk-master/npm-debug.log
```

- `LiskHQ/lisk/test.sh` does not set the `NODE_ENV` variable with a value of `test`.

### Steps to reproduce

On a machine which has not executed `LiskHQ/lisk/test.sh` before, run the following command:

```
npm run test untagged all
```

--- a/test.sh
+++ b/test.sh
@@ -63,7 +63,7 @@ reset_data() {
 
 setup_suite() {
 	# Stop lisk
-	pm2 --silent stop app.js
+	pm2 --silent stop app.js || true
 
 	# Reset data to clean state
 	reset_data

--- a/test.sh
+++ b/test.sh
@@ -69,7 +69,7 @@ setup_suite() {
 	reset_data
 
 	# Start lisk
-	pm2 --silent start app.js
+	NODE_ENV="test" pm2 --silent start app.js
 	sleep 3
 }
 

